### ISSUE TITLE
Additionnal products selected for Windows Server 2016 and 2019

### ISSUE THREAD

**Issue by abreniaux** [TraGicCode/tragiccode-wsusserver#52]
Hello guys,

I use the last 1.1.4 module and I noticed that when I select "Windows Server 2016 and Windows Server 2019" as products on my puppetfile, I have got two additionnal"non wanted" products selected on my WSUS Products console:
Developper Tools, Runtimes, and Redistribuables / Windows Server 2016
Developper Tools, Runtimes, and RedistribuablesWindows Server 2019

Do you know how to fix that issue?
Thanks a lot for your time, I appreciate.
Regards.

**Comment by TraGicCode** [comment]
Hey @abreniaux ,

I believe this is the same issue as in #51 

you will need to downgrade to 1.1.3 until the above issue gets resolved.  I unfortunately do not work with puppet much anymore so i'm relying on the community to fix it for now.

**Comment by TraGicCode** [comment]
Hey @amr952 and @mike406,

Any idea about this one?

**Comment by mike406** [comment]
It is not a bug. The PR that was merged selects ALL products that match the given string. That is the point as without selecting all matches it will only select one but it is not always the one you want. How is it supposed to know if you mean "Windows Server 2016" in Developer Tools or "Windows Server 2016" in Windows? It can't possibly know what you are thinking, so it picks all by default. You simply need to uncheck the ones you don't want in the WSUS console and it won't re-add them.

Please refer to bug https://github.com/TraGicCode/tragiccode-wsusserver/issues/46 for background info on why this change was implemented. 

In short, there is no perfect fix, it's really Microsoft's fault for using non-unique product names in different categories. But this seems to be the best compromise as you are at least able to override the default behavior by unchecking unwanted products. In the previous version of the module, you could not override it and it would ALWAYS select one match and one match only.
